_layouts/report.md

```diff
@@ -127,7 +127,7 @@
                     </thead>
                     <tbody class="card activity">
                         {% for commits in page.activity %}
-                        {% for c in commits %} 
+                        {% for c in commits %}
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.